xcube/core/store/fs/store.py

```diff
@@ -581,6 +581,8 @@ def _generate_data_ids(self,
                            return_tuples: bool,
                            current_depth: int):
         root = self.root + ('/' + dir_path if dir_path else '')
+        if not self.fs.exists(root):
+            return
```

Comment:
Why return? where is the directory created in case it does not exist?

Reply:
Return because no data ids exist